components/dropdown/dropdown-content-mixin.js

```diff
@@ -62,7 +62,7 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 				attribute: 'no-mobile-close-button'
 			},
 			/**
-			 * Override default mobile dropdown style. Specify one of 'left' or 'right'.
+			 * Override default mobile dropdown style. Specify one of 'left','right', or 'bottom'.
```

Comment:
I believe the way to get these options to show up properly in the new Daylight site is:

```suggestion
			 * Mobile dropdown style.
			 * @type {'left'|'right'|'bottom'}
```

Should we default this to something like `'none'` instead of `undefined`?

Reply:
I was thinking to leave this as undefined. This way, we prevent sprouting an attribute that says mobile-tray="none" that the consumer was not aware of.